From 42a00a0fb4c69d940ac5f6b08a57e045e14f22f7 Mon Sep 17 00:00:00 2001
From: Joseph Myers <joseph@codesourcery.com>
Date: Fri, 7 Aug 2020 14:38:12 +0000
Subject: [PATCH 014/267] Use Linux 5.8 in build-many-glibcs.py.

This patch makes build-many-glibcs.py use Linux 5.8.

Tested with build-many-glibcs.py (host-libraries, compilers and glibcs
builds).
---
 scripts/build-many-glibcs.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/build-many-glibcs.py b/scripts/build-many-glibcs.py
index efd9a9e6dc..325591b2c6 100755
--- a/scripts/build-many-glibcs.py
+++ b/scripts/build-many-glibcs.py
@@ -751,7 +751,7 @@ class Context(object):
                             'gcc': 'vcs-10',
                             'glibc': 'vcs-mainline',
                             'gmp': '6.2.0',
-                            'linux': '5.7',
+                            'linux': '5.8',
                             'mpc': '1.1.0',
                             'mpfr': '4.1.0',
                             'mig': 'vcs-mainline',
-- 
2.29.0

